azure-queue-storage/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.jclouds.labs</groupId>
+        <artifactId>jclouds-labs</artifactId>
+        <version>2.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>azure-queue-storage</artifactId>
+    <name>jclouds Azure Storage queue provider</name>
+    <description>jclouds components to access Azure queue storage Service</description>
+    <packaging>bundle</packaging>
+
+    <properties>
+    <test.azure-queue-storage.identity>${test.azure-queue-storage.identity}</test.azure-queue-storage.identity>
+    <test.azure-queue-storage.credential>${test.azure-queue-storage.credential}</test.azure-queue-storage.credential>
```

Comment:
These should not be self-referential.  Instead set to `FIXME_IDENTITY` and `FIXME_CREDENTIAL` as in the b2 provider.  This will resolve some of your CloudBees issues.